core/src/main/csharp/ch/cyberduck/core/CredentialManagerPasswordStore.cs

```diff
@@ -209,29 +224,42 @@ public override void save(Host bookmark)
             }
         }
 
-        private static Uri ToUri(Host bookmark)
+        private static Uri[] ToUri(Host bookmark)
```

Comment:
Use `List<Uri>`.
It is in private use, doesn't need allocating another array from a Collection.